x/evm/types/msg.go

```diff
@@ -331,7 +331,8 @@ func (msg MsgEthereumTx) AsMessage(signer ethtypes.Signer, baseFee *big.Int) (co
 // GetSender extracts the sender address from the signature values using the latest signer for the given chainID.
 func (msg *MsgEthereumTx) GetSender(chainID *big.Int) (common.Address, error) {
 	signer := ethtypes.LatestSignerForChainID(chainID)
-	from, err := signer.Sender(msg.AsTransaction())
+	tx := msg.AsTransaction()
```

Comment:
we better to fill the `tx.from` field using `msg.From` here.

Reply:
yes? ethtyes.Sender set in [tx.from.Store](https://github.com/ethereum/go-ethereum/blob/v1.10.26/core/types/transaction_signing.go#L146)

Reply:
but the tx is temporary object, cache sender inside is not useful.

Reply:
https://github.com/crypto-org-chain/ethermint/pull/227
I did a simple fix here.